spec/models/author_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe Author, type: :model do
```

Comment:
add tests like {.is_expected.to have_db_column }

Reply:
fixed